src/toxcore/dht/kbucket.rs

```diff
@@ -115,6 +100,8 @@ pub struct Bucket {
     pub capacity: u8,
     /// Nodes that bucket has, sorted by distance to PK.
     pub nodes: Vec<DhtNode>,
+    /// Flag for ipv6 enabled or not
+    pub is_ipv6_enabled: bool,
```

Comment:
Did you use this flag anywhere?

Reply:
https://github.com/tox-rs/tox/blob/7cdd81e5605f3e1f3c6d386babfdf99728e3480b/src/toxcore/dht/kbucket.rs#L306

Reply:
You may pass it as arg like you did before in `.to_packed_node(self.is_ipv6_enabled))`